test/specs/core.specs.js

```diff
@@ -47,8 +47,6 @@ describe('Core specs', function() {
         e.preventDefault()
       })
     })
-    // start router
-    route.start()
```

Comment:
@GianlucaGuarini why you remove this line?
This causes "detects link clicked" failure.